packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -519,12 +519,12 @@ test.serial('writes pool metrics to vstorage', async t => {
 test.serial('distributes fees per BLD staker decision', async t => {
   const { walletFactoryDriver: wd, buildProposal, evalProposal } = t.context;
 
-  const ContractFee = 152_000n; // see split above
-  t.is(((ContractFee - 60_000n) * 5n) / 10n, 46_000n);
+  const ContractFee = 32000n; // see split above
+  t.is(((ContractFee - 16_000n) * 5n) / 10n, 8_000n);
 
   const cases = [
-    { dest: 'agoric1a', args: ['--fixedFees', '0.06'], rxd: '60000' },
-    { dest: 'agoric1b', args: ['--feePortion', '0.5'], rxd: '46000' },
+    { dest: 'agoric1a', args: ['--fixedFees', '0.016'], rxd: '16000' },
+    { dest: 'agoric1b', args: ['--feePortion', '0.5'], rxd: '8000' },
```

Comment:
you asked about this one... yes, it looks right.

When the ContractFee from above goes down, there isn't as much in fees available to distribute, so these number have to be tweaked.

The `--fixedFees` choice is some arbitrary number below the ContractFee. Then the 2nd `rxd` is determined by half of what's left over.

Reply:
Cool thanks for explaining here